docs/extensions.rst

```diff
@@ -77,3 +77,46 @@ and treat it as if the client had made a request.
 
 The ASGI server should set the pseudo ``:authority`` header value to
 be the same value as the request that triggered the push promise.
+
+Zero Copy Send
+------------------
+
+Zero copy send allows you to send the contents of a file descriptor to the
+HTTP client with zero copy. ASGI servers that implement this extension will
+provide ``http.response.sendfile`` in the extensions part of the scope::
+
+    "scope": {
+        ...
+        "extensions": {
+            "http.response.sendfile": {},
```

Comment:
Can we change the extension identifier to be `http.response.zerocopysend`?

Reply:
This name continues from https://linux.die.net/man/2/sendfile and https://docs.python.org/3.7/library/asyncio-eventloop.html#asyncio.loop.sendfile

Reply:
I think it would be better to keep the name `sendfile`.

Reply:
I know, but there is also an `X-Sendfile` HTTP header that is commonly used, so we cannot just call it "sendfile".

Reply:
Okay, I agree that a distinction should be made.